app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
+
+### `strict_syntax`
+
+If `strict_syntax` is set to true then the answer and response must have `*` or `/` between each part of the expressions and exponentiation must be done using `**`, e.g. `a*b/c**2` is accepted but `ab/c^2` is not.
+
+If `strict_syntax` is set to false, then `*` can be omitted and `^` used instead of `**`, e.g. `a*b/c**2` and `ab/c^2` will be considered identical. In this case it is also recommended to list any multicharacter symbols expected to appear in the response as input symbols.
+
+By default `strict_syntax` is set to true.
 
 ## Examples
-*List of example inputs and outputs for this function, each under a different sub-heading*
-
-### Simple Evaluation
-
-```python
-{
-  "example": {
-    "Something": "something"
-  }
-}
-```
-
-```python
-{
-  "example": {
-    "Something": "something"
-  }
-}
-```
\ No newline at end of file
+
+Implemented versions of these examples can be found in the module 'Examples: Evaluation Functions'.
+
+### 1 Checking if a set of quantities match the Buckingham pi theorem
+
+#### a)
+
+In this example the task is: Given $U$, $L$ and $\nu$, suggest a dimensionless group.
+
+For this problem we do not need to predefine any quantities and give exact dimensions. The algorithm assumes that all symbols in the answer (that are not numbers or predefined constants such as $\pi$) are quantities and that there are no other quantities that should appear in the answer.
+
+**Note:** This means that the algorithm does not in any way check that the stated answer is dimensionless, that is left to the task author.
+
+For this example an EXPRESSION response area is used with answer set to `U*L/nu`. It is not necessary to use this specific answer, any example of a correct dimensionless group should work.
+
+With default settings it is required to put `*` (or `/`) between each part of the response and answer. To remove this requirement the grading parameter `strict_syntax` is set to false. Since `nu` is a multicharacter symbol it needs to be added as an input symbol.
+
+#### b)
+
+In this example the task is: Given $U$, $L$, $\nu$ and $f$, determine the necessary number of dimensionless groups and give one example of possible expressions for them.
+
+This task is similar to example a) with two significant differences. First, adding $f$ means that there are now two groups required, and second the task will constructed by defining the quantities and let the function compute the rest on its own instead of supplying a reference example.
+
+For this example an EXPRESSION response area is used with `quantities` set to `('U','(length/time)') ('L','(length)') ('nu','(length**2/time)') ('f','(1/time)')` and `answer` set to `-`.
+
+With default settings it is required to put `*` (or `/`) between each part of the response and answer. To remove this requirement the grading parameter `strict_syntax` is set to false. Since `nu` is a multicharacter symbol it needs to be added as an input symbol.
```

Comment:
'Evaluation Function Parameters' (not grading parameter)

Reply:
Presence of 'grading' is an artifact remaining from older version of documentation. Removed all found instances and generally removed specific references to the web client UI as it might change or the evaluation function might be used elsewhere.